behat/symfony2-extension/2.1/Makefile

```diff
@@ -0,0 +1,5 @@
+behat-init:
+	@test -f bin/behat && bin/behat --init || echo "cannot init behat"
+
+behat:
+	@test -f bin/behat && bin/behat || echo "cannot run behat"
```

Comment:
We already merged support for using behat.yaml natively in Behat (and with the automatic fallback to behat.yaml.dist, which is not supported by your alias, which *requires* overwriting the config locally). This is a matter of release Behat 3.4.0.